src/main.rs

```diff
@@ -122,6 +122,10 @@ async fn main() {
                     .and(with_private_key(private_key.clone()))
                     .and_then(sign_claims);
 
+    let health = warp::path!("health")
+                    .and(warp::get())
+                    .map(|| "OK");
```

Comment:
Any guarantee the http response status code is 200 ?

For the token, I explicitely wrote the `OK` may be something like:

```
warp::reply::with_status(
                "OK".to_string(),
                StatusCode::OK,
            )
```